clang/test/dpct/cuda-math-intrinsics.cu

```diff
@@ -2377,20 +2377,24 @@ __global__ void testUnsupported() {
   // CHECK: i = dpct::ffs<long long int>(ll);
   i = __ffsll(ll);
   // CHECK: /*
-  // CHECK-NEXT: DPCT1007:{{[0-9]+}}: Migration of __funnelshift_l is not supported.
+  // CHECK-NEXT: DPCT1017:207: The  call is used instead of the __funnelshift_l call. These two calls do not provide exactly the same functionality. Check the potential precision and/or performance issues for the generated code.
```

Comment:
The warning msg missed the new function name.
Maybe we need to create a new error index since it's not migrated to a funcition.

Reply:
+1 to refine the warnning,  consider to introdue new diagnostic message here.

Reply:
I've added a new diagnostic message for functions that are emulated using non-call expressions.